presto-verifier/src/main/java/com/facebook/presto/verifier/Validator.java

```diff
@@ -82,6 +82,8 @@
 
     private QueryResult controlResult;
     private QueryResult testResult;
+    private List<QueryResult> preQueryResults;
+    private List<QueryResult> postQueryResults;
 
     private boolean deterministic = true;
 
```

Comment:
I would change this constructor so that it delegates to the new one that you added (you'll probably need to add an extra parameter or two to your new one). That way all the logic for validating parameters will be in one place